httpbin/core.py

```diff
@@ -537,10 +537,91 @@ def generate_bytes():
             yield(bytes(chunks))
 
     headers = {'Transfer-Encoding': 'chunked',
-               'Content-Type': 'application/octet-stream'}
+               'Content-Type': 'application/octet-stream' }
```

Comment:
Why did you change this? It's not a style improvement and it has no other benefits. Please remove the trailing space.